### ISSUE TITLE
Web UI: task model Log and All Instances buttons inoperative when _not_ using Kubernetes

### ISSUE THREAD

**Issue by mjpieters** [apache/airflow#12289]
**Apache Airflow version**: 2.0.0b2

**Environment**:
- **Others**: No K8S

**What happened**:

In the The *Logs* and *All Instances* buttons in the task model in the web UI are inoperative when _not_ using Kubernetes.

**What you expected to happen**:

The buttons to have working links. 

The `dag.html` `updateModalUrls()` function [assumes that `buttons.rendered_k8s` is set](https://github.com/apache/airflow/blob/7d5d334857114903783390333e23e07bc9df2e2e/airflow/www/templates/airflow/dag.html#L428-L434), which is not the case when not using K8S.

This then results in an error and the buttons initialised after this point are not wired up:

```js
18:28:17.990 Uncaught TypeError: elm is undefined
    updateButtonUrl http://.../tree?dag_id=full_pipeline:1595
    updateModalUrls http://.../tree?dag_id=full_pipeline:1616
    call_modal http://.../tree?dag_id=full_pipeline:1665
    update http://.../tree?dag_id=full_pipeline:1427
    $/< http://.../static/dist/d3.min.js:1
    i http://.../static/dist/d3.min.js:1
    each http://.../static/dist/d3.min.js:3
    Y http://.../static/dist/d3.min.js:1
    each http://158.101.169.4:8080/static/dist/d3.min.js:3
    on http://158.101.169.4:8080/static/dist/d3.min.js:3
    update http://158.101.169.4:8080/tree?dag_id=full_pipeline:1420
    <anonymous> http://158.101.169.4:8080/tree?dag_id=full_pipeline:1339
tree:1595:7
```

The call needs to be protected by a `k8s_or_k8scelery_executor` check.

**Comment by SDubrulle** [comment]
I'm experiencing the same thing. Seems to be introduced in the latest beta as I did not have this issue when using the alpha release.

**Comment by mjpieters** [comment]
This is a duplicate of #12254, which was fixed in #12268.
